docs/options api.md

```diff
@@ -10,10 +10,10 @@ Archipelago will be abbreviated as "AP" from now on.
 ## Option Definitions
 Option parsing in AP is done using different Option classes. For each option you would like to have in your game, you
 need to create:
-- A new option class with a docstring detailing what the option will do to your user.
+- A new option class, with a docstring detailing what the option does, to be exposed to the user.
 - A `display_name` to be displayed on the webhost.
```

Comment:
Nothing actually *requires* making a `display_name` so this should probably be updated